go/consensus/tendermint/full/light.go

```diff
@@ -59,25 +60,24 @@ func (t *fullService) GetLightBlock(ctx context.Context, height int64) (*consens
 }
 
 // Implements LightClientBackend.
-func (t *fullService) GetParameters(ctx context.Context, height int64) (*consensusAPI.Parameters, error) {
-	if err := t.ensureStarted(ctx); err != nil {
-		return nil, err
-	}
-
-	tmHeight, err := t.heightToTendermintHeight(height)
+func (n *commonNode) GetParameters(ctx context.Context, height int64) (*consensusAPI.Parameters, error) {
+	tmHeight, err := n.heightToTendermintHeight(height)
 	if err != nil {
 		return nil, err
 	}
-	params, err := t.client.ConsensusParams(ctx, &tmHeight)
+	// Query consensus parameters directly from the state store, as fetching
+	// via tmcore.ConsensusParameters also tries fetching latest uncommitted
+	// block which wont work with the archive node setup.
+	consensusParams, err := n.stateStore.LoadConsensusParams(tmHeight)
 	if err != nil {
-		return nil, fmt.Errorf("%w: tendermint: consensus params query failed: %s", consensusAPI.ErrVersionNotFound, err.Error())
+		return nil, err
```

Comment:
Any reason why the error has changed from `ErrVersionNotFound`?

Reply:
Nope, changed back to `ErrVersionNotFound`